spec/expression.dd

```diff
@@ -1592,47 +1592,47 @@ S s = S(1, 2);
     Uniform construction syntax for built-in scalar types))
 
 
-$(H2 $(LNAME2 index_expressions, Index Expressions))
+$(H3 $(LNAME2 index_operations, Index Operations))
```

Comment:
This and the other LNAME2 changes would need to use LEGACY_LNAME2 to preserve the original anchor.